### ISSUE TITLE
IntoIterator for &'a CheeseSlice

### ISSUE THREAD

**Issue by inv2004** [lumol-org/soa-derive#10]
Hello,

I was trying to create trait which works with &'a CheeseVec and &'a CheeseSlice the same way.
But was a bit strange that I cannot find IntoIterator for &'a CheeseSlice.

Are there any reasons or problems why it was not implemented?

Thank you,

**Comment by inv2004** [comment]
```
impl<'a> IntoIterator for &'a CheeseSlice<'a> {
  type Item = CheeseRef<'a>;
  type IntoIter = __detail_iter_cheese::Iter<'a>;

  fn into_iter(self) -> Self::IntoIter {
    self.iter()
  }
}
```

**Comment by inv2004** [comment]
https://github.com/lumol-org/soa-derive/pull/11

**Comment by Luthaf** [comment]
Yeah, this was an oversight from me, we can add this implementation.

**Comment by Luthaf** [comment]
I just released version 0.6.1 with this change.
